code-challenges.js

```diff
@@ -58,12 +86,25 @@ const givenValue1 = 42
 const givenValue2 = 10
 // Expected output: 8
 
+// console.log(numberOfConnections.lastIndexOf(givenValue1));
+// console.log(numberOfConnections.lastIndexOf(givenValue2))
+
 // --------------------5) Create the code that will sort all the numbers in an array from largest to smallest. Use the test variables provided below.
 
 // Pseudo code:
+// input: array
+// output: array sorted largest to smallest
+// process: sorted function
```

Comment:
Your pseudo code for most of the challenges is not explaining how things are getting done, I would work on better explanations!